packages/apps/src/NodeInfo.tsx

```diff
@@ -16,8 +16,15 @@ import translate from './translate';
 
 type Props = I18nProps & {};
 
+export function isTestChain(chain?: string): boolean {
+  if (typeof chain === 'undefined') return false;
+  const re = new RegExp("(dev|loc)", "i");
```

Comment:
Does the RegEx need to be instantiated each time or can it be defined higher up?